bazelisk.go

```diff
@@ -388,6 +389,11 @@ func determineURL(fork string, version string, isCommit bool, filename string) s
 		return fmt.Sprintf("https://storage.googleapis.com/bazel-builds/artifacts/%s/%s/bazel", platforms[runtime.GOOS], version)
```

Comment:
I think when the user specifies a local mirror, we should also redirect these nightly builds to that server or alternatively just not support them and error out.

Reply:
this mean that i need to move `os.Getenv("BAZELISK_BASE_URL")` to above this `isCommit` check